br/pkg/config/kv.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 PingCAP, Inc. Licensed under Apache-2.0.
+package config
+
+import (
+	"encoding/json"
+
+	"github.com/docker/go-units"
+)
+
+type ConfigSet[T uint | uint64] struct {
```

Comment:
`ConfigSet` sounds like a set of config values. Maybe `ConfigTerm`.

Reply:
done